### PR TITLE
pkp/pkp-lib#6948 Run rename script during upgrade to 3.4

### PR DESCRIPTION
**Pull request by ajnyga**

### CODE REVIEW

File: classes/migration/upgrade/v3_4_0/I6759_RenameVariables.inc.php [hunk 1]
```diff
@@ -13,15 +13,15 @@
 
 namespace PKP\migration\upgrade\v3_4_0;
 
+use APP\core\Application;
```
Comment (asmecher) [inline classes/migration/upgrade/v3_4_0/I6759_RenameVariables.inc.php:16]: We need to standardize the order of imports etc., but I've been using something like...
```
use PKP\...;
use PKP\...;

use APP\...;
use APP\...;

use [third party classes];

// FIXME: Add namespacing
use [not yet namespaced classes];
```

It looks like you're preferring to put `APP` classes at the top, and adding some spaces between third-party imports; do you have a proposal for how to order these in general?

**Review by asmecher — APPROVED** [review]

File: classes/migration/upgrade/v3_4_0/I6759_RenameVariables.inc.php (hunk 1 above)
Comment (ajnyga) [reply classes/migration/upgrade/v3_4_0/I6759_RenameVariables.inc.php:16]: tbh that change was not supposed to be part of my pr, I will change that to the original order. No suggestion for the order, I have wondered sometimes what you prefer.

**Comment by ajnyga** [comment]
Restored the original order of imports

**Comment by asmecher** [comment]
Thanks, @ajnyga -- the test failure was unrelated (I've restarted it anyway) so if you're ready to go we can merge this. Are you ready?

**Comment by ajnyga** [comment]
go ahead, should be all there (fingers crossed)

**Comment by asmecher** [comment]
Merged -- thanks! This stuff is tricky.

**Pull request merged**